packages/gatsby/src/executors/server/server.impl.ts

```diff
@@ -119,6 +119,16 @@ function runGatsbyServe(
       { cwd: join(workspaceRoot, projectRoot) }
     );
 
+    childProcess.on('message', ({ action }: any) => {
+      if (
+        action?.type === 'LOG' &&
+        action?.payload?.text?.includes(options.host) &&
+        action?.payload?.text?.includes(options.port)
+      ) {
+        resolve(true);
```

Comment:
I believe this will stop the executor after the server is up... There might be an orphaned process so the server is still up but then the server won't close when it and when the user tries to start the serve again, it will error.

This method should return an AsyncGenerator like the other dev-server generators.

Reply:
Nevermind, I thought this was the main function. It's already an AsyncGenerator. :facepalm: